### ISSUE TITLE
Loss function 

### ISSUE THREAD

**Issue by lidadreamer** [jhwjhw0123/Imbalance-XGBoost#8]
论文中的weighted loss损失函数如何推导为一阶导和二阶导的为啥alpha 会变成alpha^y
